net.sf.VICE.json

```diff
@@ -0,0 +1,146 @@
+{
+    "app-id": "net.sf.VICE",
+    "branch": "stable",
+    "command": "x64",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "arch": {
+            "x86_64": {
+                "cflags": "-O3 -g -pipe -Wall -m64 -march=nocona -mtune=core2 -fexceptions -ffast-math -fstack-protector --param=ssp-buffer-size=4 -Wp,-D_FORTIFY_SOURCE=2"
```

Comment:
Does it really have to be that specific? Also what's the point of `-Wall`?

Reply:
How about these?

"cflags": "-O3 -g -m64 -march=nocona -mtune=core2 -ffast-math"

The selected options try to extract most out of current gen cpus while remaining compatible with 1st gen 64-bit cpus (Pentium 4/Athlon 64). However, I don't have any numbers to measure the speed difference so if you feel strongly about this, I can drop it to defaults.

Reply:
I'd rather keep it generic (and maybe switch to `-Ofast` for -zomg-optimize), but it can wait until the merge.